src/hooks/useJobs.tsx

```diff
@@ -11,17 +11,17 @@ const useJobs = () => {
   const { axiosManualOptions } = useMyAxios()
   const [, execute] = useAxios({}, axiosManualOptions)
 
-  const getJobs = () => {
-    const config: AxiosRequestConfig<Job[]> = {
+  const getJobs = (): Promise<AxiosResponse<Job[]>> => {
```

Comment:
The parameterization of the `AxiosRequestConfig` is not wrong, but it buys us nothing. I like this a lot better, though if we wanted to be pedantic we could call this `AxiosPromise<Job[]>`, which also buys us nothing over what you have here ;)